diff --git a/colorer/src/Colorer-library/src/colorer/strings/legacy/UnicodeString.h b/colorer/src/Colorer-library/src/colorer/strings/legacy/UnicodeString.h
index cc713d55..0a23d411 100644
--- a/colorer/src/Colorer-library/src/colorer/strings/legacy/UnicodeString.h
+++ b/colorer/src/Colorer-library/src/colorer/strings/legacy/UnicodeString.h
@@ -3,6 +3,7 @@
 
 #include <colorer/strings/legacy/CommonString.h>
 #include <memory>
+#include <stdint.h>
 
 class CString;
 
